pkg/registry/core/serviceaccount/storage/token.go

```diff
@@ -146,6 +152,27 @@ func (r *TokenREST) Create(ctx context.Context, name string, obj runtime.Object,
 				return nil, errors.NewBadRequest(fmt.Sprintf("cannot bind token for serviceaccount %q to pod running with different serviceaccount name.", name))
 			}
 			uid = pod.UID
+			if utilfeature.DefaultFeatureGate.Enabled(features.ServiceAccountTokenPodNodeInfo) {
+				if nodeName := pod.Spec.NodeName; nodeName != "" {
+					newCtx := newContext(ctx, "nodes", nodeName, api.SchemeGroupVersion.WithKind("Node"))
+					nodeObj, err := r.nodes.Get(newCtx, nodeName, &metav1.GetOptions{})
```

Comment:
I believe so? This getter ultimately references the `Store` set in the Node registry package/handler too. Digging through all that code, it appears that this storage interface *is* decorated with the cacher (as you say, if the watch cache is enabled).

Node objects don't use the raw storage, so I'm confident setting RV=0 would allow hitting the in memory cache. Please let me know if we can test for this or if it needs testing for however!